docs/website/docs/dlt-ecosystem/file-formats/_set_the_format.mdx

```diff
@@ -0,0 +1,23 @@
+import CodeBlock from '@theme/CodeBlock';
+
+<span>There are several ways of configuring dlt to use <code>{props.file_type}</code> file format for normalization step and to store your data at the destination:</span>
+
+1. <span>You can set the <code>loader_file_format</code> argument to <code>{props.file_type}</code> in the run command:</span>
+
+<pre>
+info = pipeline.run(some_source(), loader_file_format="{props.file_type}")
+</pre>
+
+
+2. You can set the `loader_file_format` in `config.toml` or `secrets.toml`:
+
+<pre>
+[normalize]{'\n'}
+loader_file_format="{props.file_type}"
+</pre>
+
+3. You can set the `loader_file_format` via ENV variable:
+
+```
+NORMALIZE__LOADER_FILE_FORMAT
+```
```

Comment:
now we have 4th method which is also cool: you can set it on a resource
https://dlthub.com/devel/general-usage/resource#pick-loader-file-format-for-a-particular-resource

Reply:
Oh, thanks for pointing this out, added!